src/main/scala/com/github/unchama/seichiassist/subsystems/lastquit/domain/LastQuitPersistence.scala

```diff
@@ -0,0 +1,17 @@
+package com.github.unchama.seichiassist.subsystems.lastquit.domain
+
+import java.util.UUID
+
+trait LastQuitPersistence[F[_]] {
+
+  /**
+   * 最終ログアウトを現在の日時で更新します
+   */
+  def updateLastQuitNow(uuid: UUID): F[Unit]
+
+  /**
+   * 最終ログアウト日時を取得します
+   */
+  def lastQuitDateTime(playerName: PlayerName): F[Option[LastQuitDateTime]]
```

Comment:
異なるUUIDを持つプレイヤーが同じプレイヤー名をかぶらないタイムスライスで持ち、彼らがそのタイムスライスでログインしたとすると、結果がSQLサーバーの実装依存 (どのプレイヤーが一番先頭に来るのか？その実装詳細に依存したくはない) になるのでこのインターフェースは不適切です。

```suggestion
  /**
  * 指定した[[UUID]]を持つプレイヤーの最終ログアウト日時を取得します。
  */
  def get(uuid: UUID): F[Option[LastQuitDateTime]]
```